cogs/committee_actions_tracking.py

```diff
@@ -49,6 +56,99 @@ class Status(Enum):
+
+        async for action in AssignedCommitteeAction.objects.select_related(
+            "discord_member"
+        ).filter(
+            Q(status=Status.IN_PROGRESS.value)
+            | Q(status=Status.BLOCKED.value)
+            | Q(status=Status.NOT_STARTED.value)
+        ):
+            grouped_filtered_actions[action.discord_member.discord_id].append(action)
+
+        return grouped_filtered_actions
+
+    async def _update_action_board(self) -> None:
+        """
+        Update the action board message with the current actions.
+
+        This method should be called after any action is created, updated or deleted.
+        """
+        if not settings["COMMITTEE_ACTIONS_BOARD"]:
+            return
+
+        action_board_channel: discord.TextChannel | None = discord.utils.get(
+            self.bot.main_guild.text_channels,
+            name=settings["COMMITTEE_ACTIONS_BOARD_CHANNEL"],
+        )
+
+        if not action_board_channel:
+            logger.warning(
+                "Action board channel could not be found so "
+                "the action board will not be updated."
+            )
+            return
+
+        action_board_message: discord.Message | None = await action_board_channel.history(
+            limit=1
+        ).next()
+
+        if not action_board_message or action_board_message.author != self.bot.user:
+            action_board_message = await action_board_channel.send(
+                content="**Committee Actions Tracking Board**\n"
+            )
+
+        all_actions: dict[
+            str, list[AssignedCommitteeAction]
+        ] = await self._get_incomplete_actions()
+
+        if not all_actions:
+            return
+
+        all_actions_message: str = "\n".join(
+            [
+                f"\n<@{discord_id}>, Actions:"
+                f"\n{
+                    ', \n'.join(
+                        (
+                            ':red_circle:'
+                            if action.status == Status.NOT_STARTED.value
+                            else ':yellow_circle:'
+                            if action.status == Status.IN_PROGRESS.value
+                            else ':no_entry:'
+                            if action.status == Status.BLOCKED.value
+                            else ':white_check_mark:'
+                            if action.status == Status.COMPLETED.value
+                            else ''
+                        )
+                        + ' '
+                        + f'{action.description} '
+                        + f'({AssignedCommitteeAction.Status(action.status).label})'
+                        for action in actions
+                    )
+                }"
+                for discord_id, actions in all_actions.items()
+            ],
+        )
+
+        await action_board_message.edit(
+            content=(f"**Committee Actions Tracking Board**\n{all_actions_message}"),
```

Comment:
```suggestion
            content=f"**Committee Actions Tracking Board**\n{all_actions_message}"
```
Superfluous brackets can be removed.